src/repositories/QueueRepository/__tests__/QueueRepository.int.spec.ts

```diff
@@ -208,4 +209,34 @@ describe("Queue Repository tests", () => {
     expect(DateTime.fromJSDate(playerInDb?.queueTime!).toISO()).toEqual(mockBallChaser.queueTime.toISO());
     expect(playerInDb?.team).toBeNull();
   });
+  it("check if player is in queue", async () => {
+    const mockBallChaser = BallChaserQueueBuilder.single();
+    await manuallyAddBallChaserToQueue(mockBallChaser);
+
+    const playerInQueue = await QueueRepository.isPlayerInQueue(mockBallChaser.id);
+
+    const playerInDb = await prisma.queue.count({
+      where: {
+        playerId: mockBallChaser.id,
+      },
+    });
+
+    expect(playerInDb).toEqual(1);
+    expect(playerInQueue).toEqual(true);
+  });
+  it("check if player is team captain", async () => {
```

Comment:
What about when the player is not in the queue? or when they're the captain of the other team?

Reply:
fixed in next commit.